app/sessionStore.js

```diff
@@ -675,6 +675,28 @@ module.exports.runPreMigrations = (data) => {
 }
 
 module.exports.runPostMigrations = (data) => {
+  // fingerprinting protection migration
```

Comment:
++ for aggregated migrations. We have many migration scattered around now and some of them are made by me :stuck_out_tongue_closed_eyes: